index.js

```diff
@@ -21,6 +21,9 @@ import Overlay from './components/Overlay';
 import Sheet from './components/Sheet';
 import Button from './components/Button';
 
+const androidKitKatSdkVersion = 19;
```

Comment:
What do you think about ?

```suggestion
const ANDROID_KIT_KAT_SDK_VERSION = 19;
```